README.md

```diff
@@ -204,7 +211,6 @@ There are three types of tests for Lantern and three corresponding commands:
 | `make test` | runs unit tests | 
 | `make test_int` | runs integration tests |
 |  `make test_e2e` | runs end-to-end tests |
-|`make test_all` | runs all tests and ends if any of the tests fail| 
```

Comment:
Minor thing, you can leave this and line 192 in since we decided to keep `make test_all` !

Reply:
Hmm weird, I don't know how this ended up being deleted by accident, I will fix that by adding it back in!